src/kaleidoscope/device/dygma/raise/Hand.h

```diff
@@ -0,0 +1,117 @@
+/* -*- mode: c++ -*-
+ * kaleidoscope::device::dygma::Raise -- Kaleidoscope device plugin for Dygma Raise
+ * Copyright (C) 2017-2019  Keyboard.io, Inc
+ * Copyright (C) 2017-2019  Dygma Lab S.L.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#ifdef ARDUINO_SAMD_RAISE
+
+#include <Arduino.h>
+#include "TWI.h"
+
+struct cRGB {
+  uint8_t r;
+  uint8_t g;
+  uint8_t b;
+};
+
+namespace kaleidoscope {
+namespace device {
+namespace dygma {
+namespace raise {
+
+#define LED_BANKS 9
+
+#define LEDS_PER_HAND 72
+#define LPH LEDS_PER_HAND
+#define LEDS_PER_BANK 8
+#define LED_BYTES_PER_BANK (sizeof(cRGB) * LEDS_PER_BANK)
+
+typedef union {
+  cRGB leds[LEDS_PER_HAND];
+  byte bytes[LED_BANKS][LED_BYTES_PER_BANK];
+} LEDData_t;
+
+// return what bank the led is in
+#define LED_TO_BANK(led) (led / LEDS_PER_BANK)
+
+// Same datastructure as on the other side
+typedef union {
+  struct {
+    uint8_t row: 2,
+            col: 3,
+            keyState: 1,
+            keyEventsWaiting: 1,
+            eventReported: 1;
+  };
+  uint8_t val;
+} key_t_raise;
```

Comment:
does this union really still get used? It dates from before we sent the whole key report as one keydata_t.

Reply:
Nope, it's unused. Thanks for catching it!